_docs_integrate/propose-attribute-to-peer.md

```diff
@@ -38,22 +38,23 @@ The Sender wants to propose an Attribute to the Recipient. To do this, the Sende
+| IdentityAttribute                                                                                       | Recipient   |     ✓     | `USER_DECISION` | Example: A company sends the name and the address of a new customer to them during their onboarding process.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| IdentityAttribute                                                                                       | Third party |     ✗     | `N/A`           | It makes no sense to propose an IdentityAttribute to the Recipient that is owned by a third party.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| RelationshipAttribute that exists in the context of a Relationship between the Sender and the Recipient | Sender      |     ✗     | `N/A`           | It makes no sense for the Sender to propose RelationshipAttributes to the Recipient that are owned by itself. Instead, it may be appropriate to proceed as described in the documentation for the [Create Attribute for peer]({% link _docs_integrate/create-attribute-for-peer.md %}) or [Read Attribute from peer]({% link _docs_integrate/read-attribute-from-peer.md %}) scenario and use the [CreateAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#createattributerequestitem) or the [ReadAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#readattributerequestitem) accordingly. |
```

Comment:
RelationshipAttribute that exists in the context of **the** Relationship between the Sender and the Recipient

Reply:
I think I find the wording a little too strong, because the Sender and the Recipient may not have a Relationship with each other from the beginning. Since there are several possible third parties, the phrase "RelationshipAttribute that exists in the context of **a** Relationship between the Recipient and a third party" must also be used in other "Combinations and usage scenarios" tables from other guides, which is why it would perhaps be more consistent to stick with "a" instead of "the".

Nevertheless, we can change it if you are sure that it would be more correct to use "the" instead of "a"!

Reply:
I'd use "the", since there cannot be multiple Relationships between the Sender and the Recipient.